.travis.yml

```diff
@@ -91,11 +91,20 @@ before_install:
     - pwd
     - mkdir /tmp/cozy-desktop
     - ln -s /tmp/cozy-desktop tmp
+    - npm install -g yarn
+    - cd gui
+    - yarn
+    - cd ..
 
 script:
     - npm run build
     - npm run lint
     - ./scripts/coverage.sh npm run test
+    - cd gui
+    - npm run lint
+    - npm run build
+    - npm run pack
```

Comment:
Well, in my experience, `npm run pack` will often says OK, even when the package is broken. I still had to test manually the packages. In particular, I had some issues with leveldb (the storage part of pouchdb) that has to be recompiled with some options to work with electron. To test that, I installed the package, launched cozy-desktop-gui, and made a synchronization.